### ISSUE TITLE
Training Error.

### ISSUE THREAD

**Issue by de2s** [lpiccinelli-eth/UniDepth#113]
Hello, thank you for releasing the training code.

When running the script [train.py](https://github.com/de2s/UniDepth-Private/blob/revert_v2old/scripts/train.py), it fails at [line 521](https://github.com/de2s/UniDepth-Private/blob/503873c3a323fbcd8bfebafce45d608a3c4c18a6/scripts/train.py#L521) because there is an exception generated at [line 133](https://github.com/de2s/UniDepth-Private/blob/503873c3a323fbcd8bfebafce45d608a3c4c18a6/unidepth/models/unidepthv1/unidepthv1.py#L133C13-L133C20) of the file [unidepthv1.py](https://github.com/de2s/UniDepth-Private/blob/revert_v2old/unidepth/models/unidepthv1/unidepthv1.py), which expects the dictionary `outputs` to have a value with the key `confidence`. However, when the object `outputs` is created at [line 221](https://github.com/de2s/UniDepth-Private/blob/503873c3a323fbcd8bfebafce45d608a3c4c18a6/unidepth/models/unidepthv1/unidepthv1.py#L221), no value with that key is defined.

**Comment by lpiccinelli-eth** [comment]
Hey, thank you for the question. Which config are you using to start the training?
There could be a loss called "confidence" in the losses in the config

**Comment by de2s** [comment]
I'm using config file [config_v1_vitl14.json](https://github.com/lpiccinelli-eth/UniDepth/blob/main/configs/config_v1_vitl14.json). It does not contain the parameter 'losses'. By the way, I'm using main branch (commit 8f44e04).
